masq/src/commands/set_configuration_command.rs

```diff
@@ -154,15 +174,15 @@ mod tests {
             *transact_params,
             vec![(
                 UiSetConfigurationRequest {
-                    name: "start-block".to_string(),
-                    value: "123456".to_string()
+                    name: name[2..].to_string(),
+                    value: value.to_string(),
                 }
                 .tmb(0),
                 1000
```

Comment:
Not your fault, but this might look better as a manifest constant, if that's what the production code is using.

Reply:
Done! 👍